File: src/test/java/info/matsumana/tsujun/TsujunApplicationTest.java
package info.matsumana.tsujun;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;

import info.matsumana.tsujun.service.KsqlService;

@SpringJUnitConfig(TsujunApplication.class)
public class TsujunApplicationTest {

    @Autowired
    private KsqlService ksqlService;

    @Test
    void contextLoads() {
        assertNotNull(ksqlService);
    }

}

File: src/test/kotlin/info/matsumana/tsujun/TsujunApplicationTest.kt
package info.matsumana.tsujun

import info.matsumana.tsujun.service.KsqlService
import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.test.context.junit.jupiter.SpringJUnitConfig

@SpringJUnitConfig(TsujunApplication::class)
class TsujunApplicationTest {

    @Autowired lateinit var ksqlService: KsqlService

    @Test
    fun contextLoads() {
        Assertions.assertNotNull(ksqlService)
    }
}
